### PR TITLE
edits consistent with UI philosophy

### PR DESCRIPTION
**Pull request by gabemontero**
@spadgett @bparees PTAL

### CODE REVIEW

File: openshift/templates/rails-postgresql.json [hunk 1]
```diff
@@ -374,106 +374,100 @@
...
-      "description": "The URL of the repository with your application source code",
+      "displayName": "Git Repository URL",
+      "description": "The URL of the repository with your application source code.",
       "value": "https://github.com/openshift/rails-ex.git"
     },
     {
       "name": "SOURCE_REPOSITORY_REF",
-      "displayName": "Source repository reference",
-      "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch"
+      "displayName": "Git reference",
```
Comment (spadgett) [inline openshift/templates/rails-postgresql.json]: Git Reference

**Comment by spadgett** [comment]
One small comment, otherwise LGTM

**Comment by gabemontero** [comment]
Yep, pushed the fix for the one nit - thx

**Comment by bparees** [comment]
lgtm

**Pull request merged**